cli/ballerina-cli/src/main/resources/cli-help/ballerina-shell.help

```diff
@@ -0,0 +1,33 @@
+NAME
+       ballerina-shell - Run ballerina interactive REPL
+
+SYNOPSIS
+       bal shell [-d|--debug] [-f|--force-dumb]  [-t|--time-out]
+
+
+DESCRIPTION
+       Shell runs a REPL instance of ballerina to enable users to run small
+       snippets to code to check their outputs. Please not that this is an
+       experimental feature.
+
+       Debug messages can be enabled via -d parameter.
+
+OPTIONS
+       -d
+           Enable debug mode from the beginning. Not enabled by default.
+       -f
+           Force dumb terminal mode. Not dumb by default.
+       -t <time-out-duration>
+           Change tree parsing timeout. Default is 100ms. Value should be
+           given in milliseconds. Increase in slow devices.
```

Comment:
Do we need "Increase in slow devices."? Not sure if it belongs in the help text.

Reply:
I'll remove it. I included this because the timeout *can* cause some problems in old hardware.